fs-diff-test-utils/privilaged-pod.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
```

Comment:
This is useful for CI testing, but not useful here.  We should remove it from this commit.